lib/logflare/backends/adaptor/tcp_adaptor/syslog.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Logflare.Backends.Adaptor.TCPAdaptor.Syslog do
+  @moduledoc """
+  Implementation of [RFC5424 The Syslog Protocol][]
+
+  It uses [octet-counting framing][].
+
+  [RFC5424]: https://www.rfc-editor.org/rfc/rfc5424
+  [octet-counting framing]: https://www.rfc-editor.org/rfc/rfc6587#section-3.4.1
+  """
+
+  alias Logflare.LogEvent
+
+  # TODO: Change it to real value
+  @pen 62137
```

Comment:
This need to be updated as soon as Supabase or Logflare will have assigned [Private Enterprise Number](https://www.iana.org/assignments/enterprise-numbers/) from IANA.